tests/NuGet.Services.AzureSearch.FunctionalTests/NuGet.Services.AzureSearch.FunctionalTests.csproj

```diff
@@ -73,6 +74,9 @@
     <PackageReference Include="NuGet.Services.Configuration">
       <Version>2.43.0</Version>
     </PackageReference>
+    <PackageReference Include="NuGet.Versioning">
```

Comment:
Can you check the product code solution and use the NuGet.* version used in those graphs? Having latest here is irrelevant since the version logic almost never changes.